docs/source/using/addons.adoc

```diff
@@ -157,6 +157,18 @@ Multiple variables need to be comma separated.
 # Required-Vars: PROJECT_USER
 ----
 
+In case, you want to have a default value for variable name, you can achieve that with _Default-Envs_ metadata header.
+Multiple default values need to be comma separated.
+
+----
+# Name: acme
+# Description: ACME add-on
+# Required-Vars: PROJECT_USER
+# Default-Envs: PROJECT_USER=john
```

Comment:
I somehow do not find the name `Default-Envs` intuitive. What about `Default-values`?

Reply:
Default-Var-Values, Default-Values does not give any hint for what the defaults are. In the context of add-ons we talk about variable interpolation and required variables, so imo we are talking about default variable values here.